### ISSUE TITLE
 [Dropdown] Luminosity ratio is less than 3:1 for already selected list item present under drop downs.

### ISSUE THREAD

**Issue by saisubrahmanya5699** [microsoft/fluentui#13835]
### Environment Information

- **Package version(s)**: (fill this out) Latest
- **Browser and OS versions**: (fill this out) Chrome

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/bssswamyberi/pen/BajwLme

### Describe the issue:

Go to codepen link as provided above
Please select one option in the dropdown 
Verify luminosity ratio is less than 3:1 for selected list item present under drop down.
![Luminosity ratio is less than 31 for selected list item](https://user-images.githubusercontent.com/48436352/85974038-05473b80-b9f2-11ea-9861-40c4fd148752.jpg)

### Actual behavior:

Luminosity ratio is less than 3:1 for already selected list item i.e - 1.19:1

### Expected behavior:

Luminosity ratio should be greater than or equal to 3:1 for already selected list item.

**Comment by Anand-Malli** [comment]
Dropdown items on hover/focus selection color does not maintain 3:1 color contrast ratio between on hover/focus background color vs normal background color.

MAS 1.4.11 guideline is explained here: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

Adding latest codepen link along with screenshot
codepen - https://codepen.io/jgage176/pen/JjyOzKK

![image](https://user-images.githubusercontent.com/65944118/142204141-00868bcd-c662-4910-b804-d8b752ba06a0.png)
